src/translations/en.json

```diff
@@ -5571,11 +5571,18 @@
         },
         "core-config": {
           "intro": "Hello {name}, welcome to Home Assistant. How would you like to name your home?",
+          "intro_core": "We will be setting up some core configuration for your Home Assistant instance now, you can always change this later in the configuration panel.",
           "intro_location": "We would like to know where you live. This information will help with displaying information and setting up sun-based automations. This data is never shared outside of your network.",
-          "intro_location_detect": "We can help you fill in this information by making a one-time request to an external service.",
+          "location_address": "Address search is powered by {openstreetmap} ({osm_privacy_policy}).",
+          "osm_privacy_policy": "Privacy policy",
+          "title_location_detect": "Do you want us to detect your location?",
+          "intro_location_detect": "We can detect your location by making a one-time request to an external service.",
+          "intro_core_config": "We filled out some details about your location, please check if they are correct and continue.",
           "location_name": "Name of your Home Assistant installation",
           "location_name_default": "Home",
+          "address_label": "Search address",
           "button_detect": "Detect",
+          "button_search": "Search",
```

Comment:
Use `ui.common.search` instead.  We already have several other duplicates of "Search".